working parts list.csv

```diff
@@ -1,24 +1,24 @@
-Part Name,Quantity,Category,Link,Note,Unit Cost (USD),Total Cost (USD)
-C-Clamp,1,Structure/Motion,https://amzn.to/3c4k5Et,Any small c-clamp will work,$15.10,$15.10
-GT2-2M Timing Belt (By the Foot),2,Structure/Motion,https://openbuildspartstore.com/gt2-2m-timing-belt-by-the-foot/?ref=_2E7m4SXRIy8t,,$2.49,$4.98
-Horizontal Rail (250mm),1,Structure/Motion,https://openbuildspartstore.com/v-slot-20x20-linear-rail/?ref=_2E7m4SXRIy8t,,$3.29,$3.29
-Vertical Rail (250mm),1,Structure/Motion,https://openbuildspartstore.com/v-slot-20x40-linear-rail/?ref=_2E7m4SXRIy8t,,$3.99,$3.99
-Xtreme Solid V Wheel Kit,6,Structure/Motion,https://openbuildspartstore.com/xtreme-solid-v-wheel-kit/?ref=_2E7m4SXRIy8t,Black Delrin or V-wheel kits can be substituted,$6.99,$41.94
-LED,1,Lighting,https://www.mouser.com/ProductDetail/997-L2C5-35901203E09,,$5.60,$5.60
-LED Housing,1,Lighting,https://www.mouser.com/ProductDetail/Ledil/FP15948_HEKLA-H?qs=0lQeLiL1qybeRTKyjbFVcA%3D%3D,,$5.39,$5.39
-LED Power Supply,1,Lighting,https://www.mouser.com/ProductDetail/MEAN-WELL/LDD-300H?qs=erfQA2AIGbWxpHw6r2bgIA%3D%3D,,$9.95,$9.95
-Lens,1,Lighting,https://www.mouser.com/ProductDetail/928-CP16109,,$5.14,$5.14
-6mm Idler Pulley (Pack of 10),1,Hardware,https://amzn.to/3AthhKy,,$11.99,$11.99
-Aluminum Spacers (40mm) (Pack of 10),1,Hardware,https://openbuildspartstore.com/aluminum-spacers-10-pack/?ref=_2E7m4SXRIy8t,Connects vertical rail to base,$7.89,$7.89
-Aluminum Spacers (6mm) (Pack of 10),1,Hardware,https://openbuildspartstore.com/aluminum-spacers-10-pack/?ref=_2E7m4SXRIy8t,Four needed for mounting wheels,$3.39,$3.39
-Assorted metric screws and nuts,1,Hardware,https://amzn.to/3camT2N,"Used to secure belt clips and pulleys (don't get this unless you need the m2 and m3 screws, it's one of the most expensive items on the list)",$32.98,$32.98

```

Comment:
Thanks for the links. I was searching for European / German suppliers as well.

You have to re-add the quotation marks, otherwise, the comma in the description messes with the CSV columns.

Reply:
> 
@clemensul On a quick note, mouser is not allowed to ship 709-LDD-300H (the led power supply) to private persons in germany due to some CE restrictions.

Reply:
Thats why I didnt include it in the European parts list, also because a replacement isnt that easy to find with the same specs on Mouser due to the high voltage LED.

Reply:
Thank you!

Reply:
@clemensul Had to update a few of the new parts. I did my best to find an EU version for each (for Mouser parts) and I used the eu.mouser url, is that cool? 🇳🇱

By the way, is there a European equivalent to something like McMaster Carr? (https://www.mcmaster.com/)

Reply:
@FinstererFred Thanks for the hint, I found out as well when I was going to checkout. 😞 
But since the delivery cost is pretty high for Mouser I am searching for an alternative anyways.

@stevenbennett I ordered all mechanical parts yesterday. I will wait for your LED update until I progress with the electronics. Thanks for the awesome project! 😃